known_friends.md

```diff
@@ -5,6 +5,8 @@ To get to know how we define "adoptions" and see how to submit a feature, please
+| IDS Broker Extension                               | EDC connector extension that can register offered assets in form of IDS Resources to a central IDS Metadata Broker.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         | [contact@sovity.de](mailto:contact@sovity.de)                                                                                                | [Link to Repo](https://github.com/sovity/edc-extensions) |
+| EDC GUI                                            | EDC Data Dashboard extended by [sovity](https://sovity.de/) and licensed under Apache 2.0. New asset metadata, multiple configurations, form validation, and many other features to come. Kept compatible with open-source EDC Connector interfaces.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        | [contact@sovity.de](mailto:contact@sovity.de)                                                                                                | [Link to Repo](https://github.com/sovity/edc-ui) |
```

Comment:
> New asset metadata, multiple configurations, form validation, and many other features to come. Kept compatible with open-source EDC Connector interfaces.

You are free to describe your solution and its features as the other posts do. But please leave out the marketing statements about possible future features. That's not what the known_friends list is for.

Reply:
Then probably best to remove the sentence: _New asset metadata, multiple configurations, form validation, and many other features to come._ (@richardtreier ?)

Reply:
Maybe:
```
EDC Data Dashboard customized by [sovity](https://sovity.de/). It integrates the open-source EDC Connector interfaces to manage asstes, set configurations, etc.
```

Reply:
Hmm, I'd prefer something a little more specific such as: 

Maybe:

`EDC Data Dashboard customized by [sovity](https://sovity.de/). It integrates the open-source EDC Connector interfaces while adding opinionated asset fields, adding form validation / design / UX improvements and configuration profiles.`